.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ on:
   - cron:  '0 2 * * *'
 
 env:
-  vcpkg_robotology_TAG: v0.8.0
+  vcpkg_robotology_TAG: v0.10.1
```

Comment:
I would also remove `Qhull_TAG: 2020.2` since it is not used anymore.

Reply:
It is still used in Ubuntu: https://github.com/ami-iit/bipedal-locomotion-framework/blob/traversaro-patch-3/.github/workflows/ci.yml#L460 .

Reply:
You're right so why it has been removed from here: https://github.com/ami-iit/bipedal-locomotion-framework/pull/565/files#r978404248